frontend-react/src/components/ReportStreamHeader.tsx

```diff
@@ -110,9 +110,6 @@ export const ReportStreamHeader = () => {
     ];
 
     let itemsMenu = [
-        <Link href="/daily" id="daily" className="usa-nav__link">
```

Comment:
Originally - I constructed the full menu then used slice to remove the elements

Reply:
@Bisonburger ah, i see, we probably don't need the slicing out anymore because of this line
https://github.com/CDCgov/prime-reportstream/blob/62071be985a37d1c48919fc4b91e01089290d004/frontend-react/src/components/ReportStreamHeader.tsx#L135
i can update it to check `authState.isAuthenticated` like how you have the `.slice` below :slightly_smiling_face: